src/Html.php

```diff
@@ -1489,6 +1489,7 @@ public static function generateMenuSession($force = false)
                             if (is_array($val)) {
                                 foreach ($val as $k => $object) {
                                     $menu[$key]['types'][] = $object;
+                                    $menu[$key]['icon']    = $object::getIcon();
```

Comment:
This method may be missing if object it not extending CommonDBTM.
```suggestion
                                    if (method_exists($object, 'getIcon')) {
                                        $menu[$key]['icon']    = $object::getIcon();
                                    }
```